src/zc/FileStorage/__init__.py

```diff
@@ -271,12 +271,8 @@ def _txn_find(self, tid, stop_at_pack):
 
 sys.path[:] = %(syspath)r
 
-if sys.version_info.major == 2:
-    import cPickle as pickle
-else:
-    import pickle
-
 import zc.FileStorage
+from zodbpickle import pickle
```

Comment:
Is the import from line 27 done here again or do I miss something?

Reply:
Yes. zodbpickle is used instead of pickle

Reply:
@icemac: you're looking at a large string literal here, called `pack_script_template`.